lasagne/layers/helper.py

```diff
@@ -251,7 +251,7 @@ def count_params(layer):
     """
     params = get_all_params(layer)
     shapes = [p.get_value().shape for p in params]
-    counts = [np.prod(shape) for shape in shapes]
+    counts = [np.prod(shape) if shape != () else 1 for shape in shapes]
```

Comment:
You know what? I need more coffee. I was writing the wrong thing in my command line.